src/pages/settings/Payments/TransferBalancePage.js

```diff
@@ -166,70 +167,84 @@ class TransferBalancePage extends React.Component {
         const isButtonDisabled = !isTransferable || !selectedAccount;
         const errorMessage = !_.isEmpty(this.props.walletTransfer.errors) ? _.chain(this.props.walletTransfer.errors).values().first().value() : '';
 
+        const shouldShowTransferView =
```

Comment:
NAB: Maybe rename this to `canTransferBalance`?

Reply:
Seems lint is creating the issue(`Do not use negated variable names.`) for this variable name.

Reply:
Weird. Ok we can leave it as is.